src/webapp/rg_reader.py

```diff
@@ -69,21 +69,14 @@ def get_topology(indir="../topology", contacts_data=None, strict=False):
 
     skip_msg = "skipping (non-strict mode)"
 
-    for facility_path in root.glob("*/FACILITY.yaml"):
-        name = facility_path.parts[-2]
-        facility_data = load_yaml_file(facility_path)
+    for facility_path in root.glob("*"):
+        name = facility_path.parts[-1]
+        facility_yaml_path = facility_path / 'FACILITY.yaml'
+        facility_data = load_yaml_file(facility_yaml_path) if facility_yaml_path.exists() else {}
         id_ = gen_id_from_yaml(facility_data or {}, name)
         topology.add_facility(name, id_)
     for site_path in root.glob("*/*/SITE.yaml"):
         facility, name = site_path.parts[-3:-1]
-        if facility not in topology.facilities:
-            msg = f"Missing facility {facility} for site {name}.  Is there a FACILITY.yaml?"
-            if strict:
-                raise TopologyError(msg)
-            else:
-                log.error(msg)
-                log.error(skip_msg)
-                continue
```

Comment:
```suggestion
        assert facility in topology.facilities, f"Missing facility {facility} for site {name}"
```
This should basically no longer happen so change it to an assertion.
(Unfortunately github tells me you can't apply suggestions on deleted lines)